src/librustc_expand/expand.rs

```diff
@@ -18,10 +18,11 @@ use rustc_attr::{self as attr, is_builtin_attr, HasAttrs};
 use rustc_errors::{Applicability, FatalError, PResult};
 use rustc_feature::Features;
 use rustc_parse::configure;
-use rustc_parse::parser::module::{parse_external_mod, push_directory};
+use rustc_parse::parser::module::{
+    parse_external_mod, push_directory, Directory, DirectoryOwnership,
```

Comment:
Nit: could you avoid multi-line imports?

Reply:
Already done in a later commit. :)